language/diem-framework/modules/0L/TowerState.move

```diff
@@ -250,8 +296,8 @@ module TowerState {
       // Check vdf difficulty constant. Will be different in tests than in production.
       // Skip this check on local tests, we need tests to send differentdifficulties.
       if (!Testnet::is_testnet()){
-        assert(&proof.difficulty == &Globals::get_vdf_difficulty(), Errors::invalid_argument(130105));
-        assert(&proof.security == &Globals::get_vdf_security(), Errors::invalid_state(130106));
+        assert(&proof.difficulty == &Globals::get_vdf_difficulty_baseline(), Errors::invalid_argument(130105));
+        assert(&proof.security == &Globals::get_vdf_difficulty_baseline(), Errors::invalid_state(130106));
```

Comment:
Should this be get_vdf_security_baseline() instead of difficulty?

Reply:
Nice catch! Yes, but it was fixed in a subsequent PR.

The final is: https://github.com/OLSF/libra/blob/c6d6c65b8739cccb4b29152cc52ca9197d88fcb2/language/diem-framework/modules/0L/TowerState.move#L298